common/data_refinery_common/models/models.py

```diff
@@ -926,12 +925,7 @@ def get_samples(self):
 
     def get_experiments(self):
         """ Retuns all of the Experiments objects in this Dataset """
-
-        all_experiments = []
-        for experiment in self.data.keys():
-            all_experiments.append(experiment)
-        all_experiments = list(set(all_experiments))
-
+        all_experiments = self.data.keys()
```

Comment:
I'm not sure why this was changed?

Reply:
I'm not sure what was the initial reason, seems that it was added in https://github.com/AlexsLemonade/refinebio/pull/299 although it's not used there. I guess it's a good helper method to have, since the dataset is stored in a `JSONField`. 

Here it's being used in a serializer when the `/?details` parameter was introduced.
https://github.com/AlexsLemonade/refinebio/pull/517/files#diff-415cd6002ae0888c9bee613a31905f87R508

Reply:
@Miserlou is there anything you don't like about these helpers? Should we move them to a different place?

Reply:
Since the data is in a JSONField, I dont' know how valid that JSON has to be - is it still "valid" if it has two keys of the same value? etc. I don't think it's hugely important in normal usage.

Reply:
but the `JSONField` is deserialized with a `dict` right? it shouldn't have two keys with the same value.